libraries/testkit/cluster.py

```diff
@@ -145,7 +145,7 @@ def reset(self, sg_config_path):
         # Add configuration to run with xattrs
         if self.xattrs:
             playbook_vars["autoimport"] = '"import_docs": "continuous",'
-            playbook_vars["xattrs"] = '"enable_extended_attributes": true'
+            playbook_vars["xattrs"] = '"enable_shared_bucket_access": true,'
```

Comment:
comma is needed. In the json, {{ xattrs }} gets replaced with "enable_shared_bucket_access": true, . Without comma, it would be an invalid json.